infohud.asm

```diff
@@ -567,6 +583,39 @@ charge_done:
 +	JMP Etanks	
 	;--/xfactor timer--
 
+	;--dash counter-- ;
+	DashCounter:
+	CLC
+	LDA $0B3F : AND #$00FF : CMP $7FFB38 : BEQ + : STA $7FFB38 : JSR Hex2Dec : LDX #$008A : JSR Draw3
+	LDX #$008A : JSR CheckLeadZero : LDA $7EC68E : CMP #$0057 : BNE + : LDA #$0C09 : STA $7EC68E
```

Comment:
At the moment this line is causing issues. On one build of the rom it made the shinespark counter never expire, where on another it made it crash. This is even without the Dash Counter display being used.

If we comment out this line it seems to work fine, just with leading zeroes on the dash counter.

Total suggesting switching to asar instead of xkas, because xkas could be generating broken code here.